.github/workflows/ci.yml

```diff
@@ -179,10 +179,10 @@ jobs:
       - name: Coverage
         if: matrix.coverage && success()
         run: |
-          sudo apt-get install -y python3-setuptools python3-wheel
+          #sudo apt-get install -y python3-setuptools python3-wheel
           # needed for https support for coveralls building cffi only works with gcc, not with clang
-          CC=gcc pip3 install --user cpp-coveralls pyopenssl ndg-httpsclient pyasn1
-          ~/.local/bin/coveralls -b "${SRCDIR}" -x .xs -e "${SRCDIR}"/if_perl.c -e "${SRCDIR}"/xxd -e "${SRCDIR}"/libvterm --encodings utf-8 -t "${{ secrets.GITHUB_TOKEN }}"
+          #CC=gcc pip3 install --user cpp-coveralls pyopenssl ndg-httpsclient pyasn1
+          #~/.local/bin/coveralls -b "${SRCDIR}" -x .xs -e "${SRCDIR}"/if_perl.c -e "${SRCDIR}"/xxd -e "${SRCDIR}"/libvterm --encodings utf-8 -t "${{ secrets.GITHUB_TOKEN }}"
           (cd "${SRCDIR}" && bash <(curl -s https://codecov.io/bash))
```

Comment:
Is the `git` command available in the $PATH?
The codecov's bash script may not work correctly if git is not available.

Reply:
Ah, sorry. It seems that I commented on an outdated commit.